exercises/target/canonical-data.json

```diff
@@ -0,0 +1,65 @@
+{
+  "exercise": "target",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Retun the correct amount earnt by a dart landing in a given point in the target problem.",
+      "cases": [
+        {
+         "property": "result",
+         "description": "A dart lands outside the targetOnly a single book",
+         "input": {
+           "x":15.3,
+           "y":13.2
```

Comment:
Please have at least one space following `:`.  Some places a space is present and in others it is absent.